### DISCUSSION TITLE
Mesh Agent being detected as CoinMiner

### DISCUSSION THREAD

**Discussion by TidderM** [amidaware/tacticalrmm#1111] [Q&A] [answered]
Are any of you running ESET antivirus?  I've had the mesh agent removed on many machines as a "potential unwanted application", being detected as "coinminer".  Curious if something has changed with the mesh agent or if this is legit and someone has injected a coinminer into the agent.

**Comment by dinger1986** [comment, marked as answer]
yes known issue with ESET https://github.com/Ylianst/MeshCentral/issues/3864

**Reply by TidderM** [reply]
Thank you, I didn't search with a space in "coinminer" so I didn't find that thread.  Perfect!

**Comment by silversword411** [comment]
There's constant false positives about meshagent being flagged by av
https://github.com/Ylianst/MeshCentral/issues/3864
